pybossa/view/projects.py

```diff
@@ -3124,3 +3124,38 @@ def show_performance_stats(short_name):
                     pro_features=pro)
 
     return handle_content_type(response)
+
+
+@blueprint.route('/<short_name>/enrichment', methods=['GET', 'POST'])
+@login_required
+@admin_or_subadmin_required
+def configure_enrichment(short_name):
+    project, owner, ps = project_by_shortname(short_name)
+    ensure_authorized_to('read', project)
+    project_sanitized, owner_sanitized = sanitize_project_owner(
+        project, owner, current_user, ps)
+
+    if request.method != 'POST':
+        ensure_authorized_to('read', Project)
```

Comment:
Why do you check `read` authorization a second time?

Reply:
agree, will remove.